src/components/Card/Card.jsx

```diff
@@ -1,23 +1,8 @@
 import "./Card.scss";
 import iconArrow from "../../assets/arrow-upper-right-black.svg";
+import CategoriesMenu from '../CategoriesMenu/CategoriesMenu.jsx';
 
-const Card = ({ img, dateOrTime, title, cardInfo, links }) => {
-  typeof links === "string" ? (links = [links]) : links;
-  const linksArr = links?.map((link, index) => {
-    if (index % 2 == 0) {
-      return (
-        <li key={index} className="card__link card__link--purple">
-          {link}
-        </li>
-      );
-    } else {
-      return (
-        <li key={index} className="card__link card__link--darkpurple">
-          {link}
-        </li>
-      );
-    }
-  });
+const Card = ({ img, dateOrTime, title, cardInfo, links, condition }) => {
   return (
     <div className="card">
       <img className="card__image" src={img} alt="Card Image" />
```

Comment:
pass alt text as props for more specific description for screen readers please (line 23)